### DISCUSSION TITLE
flag opposizione in invio TS e massimo numero di invii

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#277] [Q&A] [answered]
Salve,
è possibile almeno specificare alla creazione/invio del documento il flag di opposizione all'invio TS? 

Abbiamo provato un invio massivo di circa 1000 fatture ma abbiamo ottenuto un "TOO MANY REQUESTS...", esiste un limite sul numero di invii consecutivi?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @zpj61 
il campo che cerca è extra_data.ts_opposizione
![image](https://user-images.githubusercontent.com/7254293/219424689-4065e884-d54a-46de-8930-17fb7711c0a1.png)

Per quanto riguarda gli invii, l'errore è dovuto al raggiungimento del rate limit o del limit orario:
[Limits and Quotas](https://developers.fattureincloud.it/docs/basics/limits-and-quotas)

A disposizione
Mauro

**Reply by valmoz** [reply]
Buongiotno @zpj61 abbiamo risposto alla sua domanda?
